doc/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+install(FILES piTest.1 DESTINATION usr/share/man/man1)
```

Comment:
Not part of this PR, but: Please keep in mind that this has to respect the `nodoc` option if it shoud ever be compatible with Debian policy version 4.0.0. Version 3.9.7 recommends a seperate package.
https://www.debian.org/doc/debian-policy/upgrading-checklist.html

Reply:
I wasn't aware of this, thanks for the heads up!